exporters/variables-scss/src/helpers/__tests__/colorHelper.test.ts

```diff
@@ -0,0 +1,44 @@
+import { canHexBeShortened, normalizeColor, shortHex } from '../colorHelper';
+
+const dataProviderItems = [
+  {
+    originalColor: 'ffffffff',
```

Comment:
There is no need to repeat the keys. See the original test file.

Reply:
Yes, but it seems clearer to me this way. To be honest, I don't think it's a good idea to put an array without keys in the each function. I've already discussed it within the team, that it seems confusing to us what we're actually testing if you can't see the keys.